v2/guides/blocks/session-management.mdx

```diff
@@ -0,0 +1,9 @@
+
+Besides all the authentication recipes that get initialized in the `recipeList` array, you might have noticed another call, `Session.init()`.
+This enables the session management functionalities that **SuperTokens** provides for your application.
+
+To protect your backend routes, you will have to include the `verifySession` middleware in the route definition code. 
```

Comment:
As far as I can tell nextjs example apps use `superTokensNextWrapper`. I updated the docs based on your input (with `verifySession`) but it might get confusing if what we're saying doesn't match what's in the example app